subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.h

```diff
@@ -11,16 +11,16 @@
 #define RADIO_PKT_CONF_PDU_TYPE_MSK BIT(RADIO_PKT_CONF_PDU_TYPE_POS)
 #define RADIO_PKT_CONF_PDU_TYPE_AC (0U)
 #define RADIO_PKT_CONF_PDU_TYPE_DC (1U)
-/* PHY type, two bit field */
+/* PHY type, three bit field */
 #define RADIO_PKT_CONF_PHY_POS (1U)
-#define RADIO_PKT_CONF_PHY_MSK (BIT_MASK(2U) << RADIO_PKT_CONF_PHY_POS)
+#define RADIO_PKT_CONF_PHY_MSK (BIT_MASK(3U))
 #define RADIO_PKT_CONF_PHY_LEGACY (0U)
-#define RADIO_PKT_CONF_PHY_1M (1U)
-#define RADIO_PKT_CONF_PHY_2M (2U)
-#define RADIO_PKT_CONF_PHY_CODED (3U)
+#define RADIO_PKT_CONF_PHY_1M (BIT(1U))
+#define RADIO_PKT_CONF_PHY_2M (BIT(2U))
+#define RADIO_PKT_CONF_PHY_CODED (BIT(3U))
```

Comment:
Any reason to change the values?
```suggestion
#define RADIO_PKT_CONF_PHY_1M (BIT(0U))
#define RADIO_PKT_CONF_PHY_2M (BIT(1U))
#define RADIO_PKT_CONF_PHY_CODED (BIT(2U))
```

Reply:
Thanks for spotting that wrong BIT macro values were used. The values were plain int values `1,2,3` that were wrong because PHYs are defined and used by controller as 
```
#define PHY_1M       BIT(0)
#define PHY_2M       BIT(1)
#define PHY_CODED    BIT(2)
```
hence the flags macros should reflect that.

Reply:
Macros values corrected.